models/twilio_message_enhanced.sql

```diff
@@ -90,7 +90,7 @@ final as (
         union_messages.phone_number,
         union_messages.body,
         union_messages.num_characters,
-        (union_messages.num_characters- {{ dbt.length("body_no_spaces") }}) - 1 as num_words,
+        (union_messages.num_characters- {{ dbt.length("body_no_spaces") }}) - 1 as num_words, 
```

Comment:
This is a really interesting approach! I don't have an issue with it, but we should maybe add some logic to ensure if there is not a captured word then we do not result in a negative 1 value (which is the case for all our data). I would expect all the results in our data to show 1 as the massive block of text should be registered as at least 1 word and not -1.

Reply:
Ahh, actually I didn't do the math correctly, it should be +1 and not -1. Updated